BedrockServer.cpp

```diff
@@ -239,6 +241,9 @@ void BedrockServer_WorkerThread(void* _data) {
             }
         }
 
+        // We're shutting down, do the final performance log.
+        data->server->pollTimer.stopPoll(true);
```

Comment:
Why abuse the stopPoll() function here, rather than just creating a log() function that is called from within stopPoll()?